test/.eslintrc

```diff
@@ -33,10 +33,16 @@
 
     // @easypost/eslint-config-easypost-base assumes all tests are jest. so turn these rules off
     "jest/consistent-test-it": 0,
-    "jest/valid-expect": 0
+    "jest/valid-expect": 0,
+    "global-require": "off"
   },
 
   "globals": {
     "expect": true
+  },
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
```

Comment:
Question: I don't know that setting the sourceType here like this is appropriate since our library isn't setup as a module, though I know why you did it. Is there another option to fixing the `import` lint errors instead of saying this repo is a module which it's not?